src/layout/README.md

```diff
@@ -0,0 +1,72 @@
+# Align Layout Components
+
+## Usage
+
+```javascript
+import {Row, Column} from '@uber/layout-align';
+
+// "1fr" is similar to flex-grow: 1 on the target item
+<Row items="100px 1fr 20%">
+  <Item>100px</Item>
+  <Item>Rest</Item>
+  <Item>20%</Item>
+</Row>
+
+// automatically translate gap to row-gap or column-gap
+<Column items="100px 1fr 20%" gap="24px">
+  <Item>100px</Item>
+  <Item>Rest</Item>
+  <Item>20%</Item>
+</Row>
+
+// support common CSS properties
+<Column items="100px 1fr 20%" width="360px" height="240px" color="primary200">
+  <Item>100px</Item>
+  <Item>Rest</Item>
+  <Item>20%</Item>
+</Row>
+```
+
+## Exports
+
+* `Row`
+* `Column`
+
+## `Button` API
```

Comment:
`Button` should be replaced here with `Row/Column` I guess

Reply:
Good catch! Fixed the wording.
Also added some comparison with Block component in the description above.